resources/views/groups.blade.php

```diff
@@ -0,0 +1,9 @@
+<!DOCTYPE html>
```

Comment:
usually it's the FE people that creates the blade files. This shouldn't be included in the push. same as the friends view